x/lockup/keeper/msg_server.go

```diff
@@ -32,25 +37,27 @@ func (server msgServer) LockTokens(goCtx context.Context, msg *types.MsgLockToke
 		return nil, err
 	}
 
-	// if there is an existing lock
-	if len(msg.Coins) == 1 {
```

Comment:
We're deleting this due to comment about it being guaranteed in validate basic. Confirmed this is the case!